README.md

```diff
@@ -8,6 +8,25 @@ However, on-device debugging is hard, so the engine can also be compiled to run
 
 ## Building
 
+### Using Docker (Easiest for beginners)
+
+#### Prerequisites:
+ - Working Linux/MacOS system with Docker environment
+ - Dependencies commands:
+   - docker
+   - echo
+   - whoami
+   - grep
```

Comment:
* echo
* whoami
* grep

already ship with unix systems so i find it unecessary to mention these. They're also not dependencies for Docker